cli/bpmetadata/cmd.go

```diff
@@ -0,0 +1,29 @@
+package bpmetadata
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var mdFlags struct {
+	path   string
+	nested bool
+	verfiy bool
+}
+
+func init() {
+	viper.AutomaticEnv()
+
+	MdCmd.Flags().StringVar(&mdFlags.path, "path", ".", "Path to the blueprint for generating metadata.")
+	MdCmd.Flags().BoolVar(&mdFlags.nested, "nested", false, "Flag for generating metadata for nested blueprint, if any.")
+}
+
+var MdCmd = &cobra.Command{
```

Comment:
Since this will be `bpmetadata.cmd`.
```suggestion
var cmd = &cobra.Command{
```

Reply:
You're right but it'll be `Cmd`. Changed it.